coupon_criteria_multi_product/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Coupons multi product criteria",
+    "summary": "Allows to set as promotion criteria multi-product conditions",
+    "version": "15.0.1.0.0",
+    "development_status": "Production/Stable",
+    "category": "Sale",
+    "website": "https://github.com/OCA/sale-promotion",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "maintainers": ["chienandalu"],
+    "license": "AGPL-3",
+    "depends": ["sale_coupon"],
```

Comment:
It doesn't make sense to split the module if we don't change the dependecy chain, don't you think?
```suggestion
    "depends": ["coupon"],
```

Reply:
Totally in agreement, what a cluelessness